### DISCUSSION TITLE
用LORA进行微调时报错Segmentation fault (core dumped)

### DISCUSSION THREAD

**Discussion by Wenting1227** [zai-org/ChatGLM3#935] [Q&A]
![image](https://github.com/THUDM/ChatGLM3/assets/142857717/2d7db754-ebff-45a0-a684-5f573366ddf5)
用lora进行微调时报错：Segmentation fault (core dumped)
不限制core栈的空间分配后还是会报同样的错误
各位大佬有遇到同样问题的嘛？要怎么解决呀？

**Comment by zRzRzRzRzRzRzR** [comment]
配置不够么

**Reply by Wenting1227** [reply]
更新了模型的配置文件和权重 现在好啦~感谢感谢

**Comment by Wenting1227** [comment]
更新了模型的配置文件，现在好啦




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "THUDM/ChatGLM3"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月8日(星期五) 中午12:47
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [THUDM/ChatGLM3] 用LORA进行微调时报错Segmentation fault (core dumped) (Discussion #935)





 
配置不够么
